src/main/java/uk/gov/dvsa/model/cvs/certificateData/IvaFailCertificateData.java

```diff
@@ -0,0 +1,148 @@
+package uk.gov.dvsa.model.cvs.certificateData;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
The getters and setters are used by handlebars, and for the testing.

For belts and braces I think it's best to ignore anything we are not expecting, doc-gen throws an error as it will try mapping to a key/value that doesn't exist in the model.